apps/web/src/components/workflow/preview/in-app/v2/Content.tsx

```diff
@@ -0,0 +1,219 @@
+        </Text>
+        <Text variant="main" fontWeight="strong" color="typography.text.secondary">
+          Feed
+        </Text>
+        <Text variant="main" fontWeight="strong" color="typography.text.secondary">
+          Feed
+        </Text>
+      </HStack>
+
+      {/* Content Section */}
+
+      {isPreviewLoading ? (
+        <Flex paddingBlock="1rem" gap="0.5rem" alignSelf="stretch" flexGrow={1}>
+          <Avatar radius={8}>
+            <IconPerson />
+          </Avatar>
+          <Skeletons />
+        </Flex>
+      ) : (
+        <Flex paddingBlock="1rem" gap="0.5rem" alignSelf="stretch" flexGrow={1}>
+          {/* Unread Dot */}
+          <span
+            className={css({
+              w: '0.5rem',
+              h: '0.5rem',
+              bgColor: '#369EFF',
+              borderRadius: 'circle',
+              mt: '0.125rem',
+            })}
+          />
+
+          {/* Avatar Section */}
+          {parsedPreviewState.avatar ? (
+            <Avatar src={parsedPreviewState.avatar} radius={8}>
+              <IconPerson />
+            </Avatar>
+          ) : null}
+          {/* Message Section */}
+          <Stack gap="1rem" flexGrow={1}>
+            <Stack gap="0.25rem">
+              <HStack justifyContent="space-between">
+                <Text variant="main" fontWeight="strong">
+                  {parsedPreviewState.subject}
+                </Text>
+
+                <Text variant="main" color="typography.text.secondary">
+                  5 min
+                </Text>
+              </HStack>
+              <Text variant="main" color="typography.text.secondary">
+                {parsedPreviewState.content}
+              </Text>
+            </Stack>
+
+            {/* Actions Section */}
+            {parsedPreviewState.ctaButtons && parsedPreviewState.ctaButtons.length > 0 ? (
+              <Flex gap="1rem">
+                {parsedPreviewState.ctaButtons.map((button, index) => (
+                  <button key={index} className={actionButtonRecipe({ type: button.type })}>
+                    {button.content}
+                  </button>
+                ))}
+              </Flex>
+            ) : null}
+          </Stack>
+        </Flex>
+      )}
+
+      {/* Footer */}
+      <Center gap="0.25rem" alignSelf="stretch" mt="auto">
+        <img
+          src={`/static/images/novu-gray.svg`}
+          className={css({
+            h: '0.75rem',
+            w: '0.75rem',
+            borderRadius: 'circle',
+          })}
+        />
+        <Text variant="secondary">Powered by Novu</Text>
+      </Center>
+    </div>
+  );
+}
+
+const Skeletons = () => {
+  return (
+    <Stack gap="0.75rem" w="100%">
+      <Stack gap={5}>
+        <SkeletonStyled height={14} width="35%" radius={4} />
+        <SkeletonStyled height={14} width="70%" radius={4} />
+      </Stack>
+      <SkeletonStyled height={14} width="25%" radius={4} />
+    </Stack>
+  );
+};
+
+const actionButtonRecipe = cva({
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Nice use of an atomic recipe!‏